### PR TITLE
[WIP]Add weekly updates in 2015/07/17

### PR DESCRIPTION
**Pull request by yosuke-furukawa**
We would be better to wait v2.4.0 release.

### CODE REVIEW

**Comment by yosuke-furukawa** [comment]
io.js v2.4.0 is released, update our weekly notes.

https://github.com/nodejs/evangelism/blob/2015_07_17/weekly-updates/weekly-update.2015-07-17.md

**Comment by yosuke-furukawa** [comment]
@nodejs/evangelism please review our weekly updates.

**Comment by julianduque** [comment]
LGTM

File: weekly-updates/weekly-update.2015-07-17.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+### io.js and Node.js News — July 17th
+io.js 2.4.0 is released, apigee joined our foundation and Intl WG start.
+
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-07-17.md:3]: Judging from their website, Apigee capitalizes the first letter of their name.

File: weekly-updates/weekly-update.2015-07-17.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+* `url.resolve` may transfer the auth portion of the url when resolving between two full hosts, see [#1435](https://github.com/nodejs/io.js/issues/1435).
+
+### Community Updates
+
+* NodeSource announces N|Support on [their blog post](https://nodesource.com/blog/nodesource-announces-nsupport). N|Support is a premium support offering for Node.js developers, DevOps and IT operations teams. To learn more about N|Support offerings, check out [the solution page](https://nodesource.com/products/nsupport).
+* [apigee](https://apigee.com/) is [added to the Node Foundation](https://github.com/nodejs/nodejs.org/pull/151)
+* [joyent/docker-node](https://github.com/joyent/docker-node) team is moved to [nodejs/docker](https://github.com/nodejs/docker-iojs) team. As a result, [@chorrell](https://github.com/chorrell) and [@dcrudgington](https://github.com/dcrudgington) joined [@nodejs/docker](https://github.com/orgs/nodejs/teams/docker) team.
+* [Intl](https://github.com/nodejs/intl) WG is launched. They dedicated to support and improvement of Internationalization and Localization in Node. Intl WG calls for participants in [their issue](https://github.com/nodejs/Intl/issues/5).
+
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-07-17.md:32]: Typo: `They are dedicated` rather then `They dedicated`?

**Pull request merged**